app/views/layouts/application.html.erb

```diff
@@ -6,21 +6,18 @@
     <%= javascript_include_tag "application" %>
     <%= csrf_meta_tags %>
   </head>
-  <body>
-      <% if signed_in? %>
+<body>
+    <% if signed_in? -%>
       <%= render "donations/new" %>
       <%= render "layouts/navbar" %>
     <% end %>
 
-    <% flash.each do |key, value| %>
-      <div class="alert alert-<%= key %>">
-        <%= value %>
-      </div>
+  <div id="flash">
+    <% flash.each do |key, value| -%>
+      <div id="flash_<%= key %>"><%=h value %></div>
```

Comment:
I think we want to use the Bootstrap alert styling, so we shouldn't change this.